docker-compose.yml

```diff
@@ -23,16 +24,19 @@ services:
       - NOINITDB=0
     env_file:
       - nav-variables.env
+    restart: always
 
   postgres:
-    image: "postgres:9.4"
+    image: "postgres:latest"
```

Comment:
Please keep PRs more focused. I think upgrading the PostgreSQL service is way overdue, but it seems out of scope compared to the rest of this PR.

Also, I would recommend locking it to a version number. I'm not sure how the Postgresql Dockerhub images handle it, but it wouldn't normally be a good idea to start a new version of postgresql pointing to the same data directory used by an older version.

Reply:
Will pin version to "12" rather than "latest" to avoid surprise major upgrades of Postgres. 

Currently, both "12" and "latest" is 12.1 that I've been running in prod for a while without seeing any issues.